RecoTauTag/RecoTau/src/RecoTauQualityCuts.cc

```diff
@@ -53,160 +52,6 @@ namespace reco::tau {
 
   // Quality cut implementations
   namespace qcuts {
-
```

Comment:
Yes, I confirm the qcuts implementations that I removed are not called from any code in RecoTauTag anymore. If I recall correctly, Evan originally implemented these functions and we kept them until recently, because they were still used by the HLT (with a lot of other code from RecoTauTag/TauTagTools, which became obsolete when the HLT switched to the using RecoTauBuilderConePlugin.cc instead of PFRecoTauProducer.cc)

Reply:
OK, thanks.